src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.ui;
+
+public class Formatter {
```

Comment:
May have misunderstood the question :-).

The responsibility of this class is quite a bit more. Essentially, anything to do with formatting a string (e.g. padding, adding divider etc) should be placed in this class.